src/main/java/edu/wsu/SnakePane.java

```diff
@@ -56,7 +149,7 @@ private void moveSnakeRight(List<Circle> snake){
     Circle newHead = snake.remove(snake.size() - 1);
     oldHead.setFill(bodyColor);
     newHead.setFill(headColor);
-    newHead.setCenterX((oldHead.getCenterX() + (2 * radius))%600);
+    newHead.setCenterX((oldHead.getCenterX() + (2 * radius))%width);
```

Comment:
I would remove the modulus operators from moveSnake* method -- it will make detecting the snake falling off the pane a little easier.